Cocktail_Sort/Cocktail_Sort.c

```diff
@@ -0,0 +1,67 @@
+//Cocktail sort is a variation of bubble sort that is both a stable sorting
+//algorithm and a comparison sort. The algorithm differs from a bubble sort in 
+//that it sorts in both directions on each pass through the list. 
+#include <stdio.h>  
+   
+int main()  
+{  
+    int n;
+    printf("Enter the number of elements ");
+    scanf("%d",&n);
+    int a[n];
```

Comment:
Use of better variable names than `a` and `n` will be amazing.

Reply:
@jainaman224  I have done the required changes, please review.